code/modules/hydroponics/seeds.dm

```diff
@@ -1162,7 +1162,7 @@
 	lifespan = 50
 	maturation = 3
 	production = 5
-	yield = 4
+	yield = 1
```

Comment:
This might be an accident. You reduced the yield of regular cabbage.
Did you intend to change the yield of plasma cabbage? If so, put it in the changelog under tweak.
As in:
* tweak: Plasma cabbages now make only 1 cabbage per harvest.
I'd keep the yield reduction to its own PR, though, for atomicity.